k8s/README.md

```diff
@@ -0,0 +1,438 @@
+
+It is based on the Kubernetes source at:
+  https://github.com/kubernetes/kubernetes/tree/v1.2.0
+
+For Kubernetes copyright and licensing information, see:
+  /usr/local/share/doc/kubernetes/LICENSES
+
+ver@gke-gobs-cluster-03c0208c-node-4ewy:~$ 
+```
+
+Since we haven't deployed any applications yet, there are no Pods yet:
+```
+:; kubectl get pods                                                                          
+:;
+```
+
+Although, if you may notice that there are a bunch of system processes running
+in the _kube-system_ namespace:
+```
+:; kubectl --all-namespaces=true get pods
+NAMESPACE     NAME                                                        READY     STATUS    RESTARTS   AGE
+kube-system   fluentd-cloud-logging-gke-gobs-cluster-03c0208c-node-4ewy   1/1       Running   0          5m
+kube-system   fluentd-cloud-logging-gke-gobs-cluster-03c0208c-node-m2g5   1/1       Running   0          5m
+kube-system   fluentd-cloud-logging-gke-gobs-cluster-03c0208c-node-n6m9   1/1       Running   0          5m
+kube-system   fluentd-cloud-logging-gke-gobs-cluster-03c0208c-node-vr9c   1/1       Running   0          5m
+kube-system   fluentd-cloud-logging-gke-gobs-cluster-03c0208c-node-w1k6   1/1       Running   0          5m
+kube-system   heapster-v1.0.0-oy4kv                                       1/1       Running   0          6m
+kube-system   kube-dns-v11-dsj5p                                          4/4       Running   0          6m
+kube-system   kube-proxy-gke-gobs-cluster-03c0208c-node-4ewy              1/1       Running   0          5m
+kube-system   kube-proxy-gke-gobs-cluster-03c0208c-node-m2g5              1/1       Running   0          5m
+kube-system   kube-proxy-gke-gobs-cluster-03c0208c-node-n6m9              1/1       Running   0          5m
+kube-system   kube-proxy-gke-gobs-cluster-03c0208c-node-vr9c              1/1       Running   0          5m
+kube-system   kube-proxy-gke-gobs-cluster-03c0208c-node-w1k6              1/1       Running   0          5m
+kube-system   kubernetes-dashboard-v1.0.0-72bii                           1/1       Running   0          6m
+kube-system   l7-lb-controller-v0.6.0-oa1zy                               2/2       Running   0          6m
+```
+
+You may notice that the kubernetes API server is exposed as a Service.
+```
+:; kubectl get svc
+NAME         CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
+kubernetes   10.3.240.1   <none>        443/TCP   5m
+```
+
+The `kubernetes` Service is a special immortal sort of Service that
+will come back to life if you delete it:
+```
+:; kubectl delete svc/kubernetes
+service "kubernetes" deleted
+:; kubectl get svc
+NAME          CLUSTER-IP     EXTERNAL-IP    PORT(S)           AGE
+kubernetes    10.3.240.1     <none>         443/TCP           1s
+```
+
+At this point your kubernetes cluster is at a point where we can
+deploy an application!
+
+## Gleaming the Kube ##
```

Comment:
This also really feels like the start of the document, whereas setting up your k8s cluster is really just pre-amble.  It might be better to move that into a separate file, link it from the top of this file, and jump right in.